### PR TITLE
Tower Of Hanoi

### PR DESCRIPTION
**Pull request by vanshita25**
kindly merge

### CODE REVIEW

File: towerofhanoi.c [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#include<stdio.h>
+void TowerOfHanoi(int SIZE, char c1, char c2, char c3)
+{
+    if(SIZE==1)
+    {
+        printf("%d  Disc  transferred from %c to %c \n", SIZE, c1, c3);
```
Comment (Bhavnaharitsa) [inline towerofhanoi.c]: variable name should not be all caps

File: towerofhanoi.c [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#include<stdio.h>
+void TowerOfHanoi(int SIZE, char c1, char c2, char c3)
```
Comment (Bhavnaharitsa) [inline towerofhanoi.c]: Add title and description in comments

File: towerofhanoi.c [hunk 3]
```diff
@@ -1,17 +0,0 @@
-#include<stdio.h>
-void TowerOfHanoi(int size, char c1, char c2, char c3)
-{
-    if(SIZE==1)
-    {
```
Comment (Bhavnaharitsa) [inline towerofhanoi.c]: Why did you delete file? simply could have commited it

Comment (vanshita25) [reply towerofhanoi.c]: will keep that in mind next time
